app/views/notify_user/file_uploaded.text.haml

```diff
@@ -0,0 +1,3 @@
+A file was uploaded for the following client:
+
+= client_files_url(@client)
```

Comment:
This will need to link to the window version of files, since most people don't have access to the client section.